### ISSUE TITLE
Div should not have aria-labelledby attribute

### ISSUE THREAD

**Issue by EmmaDawsonDev** [EddieHubCommunity/EventCalendar#56]
The div wrapping the calendar has an aria-labelledby attribute but this is not actually supported and can intefere with html semantics. It should be removed.

<img width="765" alt="Screenshot 2021-11-09 145645" src="https://user-images.githubusercontent.com/57045550/140937216-9961c017-d65d-4c48-89c6-487fb737539e.png">

**Comment by handikaharianto** [comment]
I think `aria-labelledby="fc-dom-1"` is generated automatically by **FullCalendar** component in **App.js**.

**Comment by EmmaDawsonDev** [comment]
Closing issue as it’s a problem with the imported library, not something in the code we can influence.
